### PR TITLE
margins in minecraft

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Move the minecraft top instructions down a little bit (top of screenshot is new, bottom is old)

image

### CODE REVIEW

File: apps/src/templates/instructions/TopInstructionsCSF.jsx [hunk 1]
```diff
@@ -46,6 +46,10 @@ const styles = {
     left: 0,
     // right handled by media queries for .editor-column
   },
+  mainCraft: {
```
Comment (islemaster) [inline apps/src/templates/instructions/TopInstructionsCSF.jsx]: That name is just _asking_ for trouble...

**Comment by islemaster** [comment]
LGTM

File: apps/src/templates/instructions/TopInstructionsCSF.jsx (hunk 1 above)
Comment (Bjvanminnen) [inline apps/src/templates/instructions/TopInstructionsCSF.jsx]: Haha, I had the same thought when I saw the PR. I think it's okay when you see the full code file tho, as it's next to a bunch of other mains.

Comment (Bjvanminnen) [inline apps/src/templates/instructions/TopInstructionsCSF.jsx]: Eh, maybe i should just change to something like mainMinecraft to be safe

**Pull request merged**